src/.buildkite/test/docker-compose.yml

```
@@ -11,6 +11,7 @@ services:
       - TEST_MYSQL_BASE_URI=mysql://root:root@mysql:3306
       - TEST_MARIADB_BASE_URI=mysql://root:root@mariadb:3306
       - TEST_POSTGRES_URI=postgres://prisma:prisma@postgres:5432/tests
+      - TEST_POSTGRES_ISOLATED_URI=postgres://prisma:prisma@postgres_isolated:5435/tests
```

```suggestion
      - TEST_POSTGRES_ISOLATED_URI=postgres://prisma:prisma@postgres_isolated:5432/tests
```
That should fix this pipeline